api/mds.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package envoy.api.v2;
+
+import "google/api/annotations.proto";
+
+// While it is not a "true" discovery service, just named it as MetricsDiscoveryService to align with xDS paradigm.
```

Comment:
Yeah, I had this initial thought for Rate Limit Service, but decided better eventually. Let's not shoe horn into the xDS space things that aren't actually discovery.

Reply:
+1. I'm going to be adding a log streaming API soon and will not be calling it xDS. But I think we need to resolve the Prometheus question before we go further on this issue. I am going to ask the Prometheus folks to hop into this issue.

Reply:
Great. Will change it.